ui/components/app/advanced-gas-fee-popover/advanced-gas-fee-input-subtext/index.scss

```
@@ -0,0 +1,17 @@
+.advanced-gas-fee-input-subtext {
+  display: flex;
+  align-items: center;
+  margin-top: 2px;
+  color: $ui-4;
+  font-size: 12px;
+
+  &__label {
+    font-weight: bold;
+    margin-right: 4px;
+  }
+
+  & img {
+    height: 16px;
+    margin-right: 12px;
```

[Should adhere to 8px grid](https://consensys.slack.com/archives/G1L7H42BT/p1637778098226300)
```suggestion
    margin-right: 8px;
```